geoh5py/workspace/workspace.py

```diff
@@ -73,6 +73,16 @@
 )
 
 
+NETWORK_DRIVES = [
+    "Egnyte Connect",
+    "One Drive",
+    "Dropbox",
+    "Google Drive",
```

Comment:
Mine is `googledrive`

Reply:
I think you must have updated it when installing, or they changed their default.  The current default is "Google Drive"

Reply:
It says Google Drive in the explorer, but the windows path is lower case. 
![image](https://github.com/user-attachments/assets/f37ae836-ba2f-4a57-b4b0-8f159f68bf9a)

Anyway, doesn't cost much to add it to the list.

Reply:
ok, added